torch/testing/_internal/common_methods_invocations.py

```diff
@@ -602,6 +602,29 @@ def sample_inputs_addmm(op_info, device, dtype, requires_grad, **kwargs):
     else:
         return (input, )
 
+def sample_inputs_addmv(op_info, device, dtype, requires_grad):
+    test_cases = [((S,), (S, M), (M,), 1, 1),
```

Comment:
See related issues; this has uncovered an addmv bug!

Luckily I don't think this needs to wait for those issues to be resolved, but there's a slight tweak that would be helpful here.

In https://github.com/pytorch/pytorch/pull/53014, @kshitij12345 updated test_variant_consistency_eager to pass a kwarg, "for_inplace_variant":

https://github.com/pytorch/pytorch/blob/0dff0d1537a3746e80dd3a3b655ec071fe3c2466/test/test_ops.py#L304

an example of this kwarg in use is here:

https://github.com/pytorch/pytorch/blob/0dff0d1537a3746e80dd3a3b655ec071fe3c2466/torch/testing/_internal/common_methods_invocations.py#L1810

This sample inputs can take the kwarg and do a few things to use it. One example is below:

```
for_inplace_variant = kwargs.get('for_inplace_variant', False)

test_cases = (
  ((S,), (S, M), (M,), 1, 1),
  ((S,), (S, M), (M,), 0.2, 0.6),
)

test_cases_with_broadcast = (
  ((1,), (S, M), (M,), 1, 1),
  ((1,), (S, M), (M,), 0.2, 0.6),
  ((), (S, M), (M,), 1, 1),
  ((), (S, M), (M,), 0.2, 0.6),
)

cases = test_cases if for_inplace_variant else (test_cases + test_cases_with_broadcast)
samples = []
for a_shape, b_shape, c_shape, alpha, beta in cases:
  a = make_tensor(a_shape, ...)
  b = make_tensor(b_shape, ...)
  c = make_tensor(c_shape, ...)
  sample = SampleInput(a, args=(b, c), kwargs={'alpha': alpha, 'beta': beta})
  samples.append(sample)

return tuple(samples)
```